proj/pred/templates/pred/help.html

```diff
@@ -43,9 +43,9 @@ <h3>1. Summary</h3><br>
             predict also the signal peptide.
             </p>
 
-            <p>
-            SCAMPI2 is supported by the <a href="https://www.egi.eu/services/cloud-compute/">EGI FedCloud</a> with the VO VO.NBIS.SE for computational resource. 
-            </p> 
+            <p>This work used the <a href="https://www.egi.eu/infrastructure/cloud/">EGI infrastructure</a>
+￼              with the support of IN2P3-IRES, INFN-CLOUD-BARI and TR-FC1-ULAKBIM.
```

Comment:
There are two weird `obj` symbol around the text. I guess it is caused by the line breaks. 
<img width="710" alt="Screenshot 2022-02-08 at 16 02 41" src="https://user-images.githubusercontent.com/5833722/153014264-2587e718-9c85-465f-aae3-24c89e5c3a60.png">